db/knex_migrations/2023-10-05-1200-add-disable-url-to-notification.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+    return knex.schema.alterTable("notification", function (table) {
+        table.boolean("disable_url").defaultTo(false).notNullable().comment("Disable URL in Discord notifications");
```

Comment:
If it is a discord only notifcation setting, it does not need a field in the db.

In fact, this field currrently is not used by frontend and backend as far as I can see below (unless I am missing something)

=> please remove this file, nuke your db and check if this still works.